caprover_api/caprover_api.py

```diff
@@ -519,11 +528,13 @@ def delete_app(self, app_name, delete_volumes: bool = False):
     @retry(times=3, exceptions=COMMON_ERRORS)
     def create_app(
         self, app_name: str,
+        project_id: str = '',
         has_persistent_data: bool = False,
         wait_for_app_build: bool = True
     ):
         """
         :param app_name: app name
+        :param project_id: project id
```

Comment:
It may be helpful to note that empty string (the default) will put the app under root (no parent project)

Reply:
Thanks added the suggested info.

Reply:
I think we are aligned with all the dashboard features now, just want to ensure all the core functionalities are in place before pushing V2.